app/css/application.css

```diff
@@ -18,8 +18,61 @@
 @import "tailwindcss/components";
 @import "tailwindcss/utilities";
 
+:root {
+    /* TODO: Rename these to match figma */
+    --c-CBC9D9: #cbc9d9;
+    --c-322C44: #322c44;
+    --c-343741: #343741;
+    --c-6D6986: #6d6986;
+    --c-6A93FF: #6a93ff;
+    --c-8480A0: #8480a0;
+    --c-EAECF3: #eaecf3;
+    --c-CBC9D9: #cbc9d9;
+    --c-26282D: #26282d;
+    --c-000000: #000000;
+    --c-FFFFFF: #ffffff;
+    --c-FBFCFE: #fbfcfe;
+    --c-282339: #282339;
+    --c-A9A6BD: #a9a6bd;
+}
+
+.theme-light {
+    --backgroundColorA: var(--c-FFFFFF);
```

Comment:
The semantic colours used within the code are then mapped to the named variables above.